src/images/projectIcons/taskcluster.svg

```diff
@@ -1 +1 @@
513 8.905M-29.158-.776l-.549 4.968M-24.856-3.51l9.93 3.379" fill="#4bacb8" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square"/><path style="marker:none" d="M-19.944 10.699a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM-17.502 3.02a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#fff" fill="#fff" overflow="visible"/><path d="M-25.738-2.042l3.35 3.133M-27.255-1.02l3.477 9.08" stroke="#fff" stroke-width="1.1297048" stroke-linecap="square" fill="none"/></g><g><path style="marker:none" d="M46.947 24c0 12.674-10.273 22.947-22.947 22.947S1.053 36.674 1.053 24 11.326 1.053 24 1.053 46.947 11.326 46.947 24z" color="#fff" fill="#222" stroke="#fff" stroke-width="2.10504" stroke-linecap="square" overflow="visible"/><path style="marker:none" d="M15.732 21.26a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM14.42 11.954a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.649 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path style="marker:none" d="M28.04 23.815a4.071 4.071 0 1 0-8.143 0 4.071 4.071 0 0 0 8.143 0z" color="#4bacb8" fill="#4bacb8" stroke="#4bacb8" stroke-width=".579044" overflow="visible"/><path style="marker:none" d="M9.364 27.185a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM39.655 22.991a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM42.935 16.98a2.824 2.824 0 1 0-5.649 0 2.824 2.824 0 0 0 5.649 0zM22.63 14.314a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM29.467 8.06a2.824 2.824 0 1 0-5.649 0 2.824 2.824 0 0 0 5.649 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M24.65 19.8l1.52-8.955M22.334 20.085L20.94 16.9M21.03 20.997l-7.395-7.088M15.66 21.895L20 22.898M19.97 24.588l-10.657 2.06M28.032 23.555l5.98-.384M37.51 18.082l-9.791 4.145" fill="none" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square"/><path style="marker:none" d="M34.465 14.304a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M29.868 16.502l-3.343 4.144" fill="none" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square"/><path style="marker:none" d="M15.964 30.906a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM17.65 38.585a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM25.487 35.637a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM43.251 29.417a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M20.562 26.046l-5.06 3.313M21.825 27.277l-5.512 8.906M23.521 27.862l-.548 4.968M27.824 25.127l9.929 3.38" fill="none" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square"/><path style="marker:none" d="M32.735 39.336a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM35.177 31.659a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M26.942 26.596l3.349 3.133M25.425 27.618l3.476 9.08" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square" fill="none"/></g></svg>
```

Comment:
Taskcluster actually has a new logo https://github.com/taskcluster/taskcluster-media/blob/main/source/logo/logo.svg

You can try using the new one and see if it looks better.

Reply:
Ok, I'll try it for sure :smile: